### PR TITLE
Add French 🥖

### PR DESCRIPTION
**Pull request by flibustier**
Et voilà…! 🥐

### CODE REVIEW

File: README-fr.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Checklist des points de sécurité les plus importants lors de la conception, du test et de la mise en production de votre API.
+
+------------------------------------------------------------------------------
+## Authentification
+- [ ] Ne pas utiliser une authentification basique http (`Basic Auth`) mais plutôt un standard d'authentification (tel que JWT, OAuth).
+- [ ] Ne pas réinventer la roue lors de `l'authentification`, `la génération de jeton`, `le stockage de mots de passe` mais utiliser les standards.
+
+### JWT (JSON Web Token)
+- [ ] Utiliser des clefs aléatoires complexes (`JWT Secret`) pour rendre les attaques par force brute difficiles.
+- [ ] Ne pas extraire l'algorithme de la payload. Imposer l'algorithme côté serveur (`HS256` ou `RS256`).
```
Comment (Sinequanonh) [inline README-fr.md]: I'd rather say `du payload`, WDYT?

File: README-fr.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# API Security Checklist
+Checklist des points de sécurité les plus importants lors de la conception, du test et de la mise en production de votre API.
+
+------------------------------------------------------------------------------
+## Authentification
+- [ ] Ne pas utiliser une authentification basique http (`Basic Auth`) mais plutôt un standard d'authentification (tel que JWT, OAuth).
+- [ ] Ne pas réinventer la roue lors de `l'authentification`, `la génération de jeton`, `le stockage de mots de passe` mais utiliser les standards.
```
Comment (Sinequanonh) [inline README-fr.md]: What about keeping the word `token` instead of `jeton` which seems overkill?

File: README-fr.md (hunk 1 above)
Comment (flibustier) [inline README-fr.md]: After debating with my colleague, you're right !

File: README-fr.md (hunk 2 above)
Comment (flibustier) [inline README-fr.md]: Yes why not, but then what about `credentials` instead of `accréditations` ?

Comment (Sinequanonh) [inline README-fr.md]: Mmh, seems trickier. Isn't it implying `identifiants`?

**Comment by Sinequanonh** [comment]
Good job! 👌

File: README-fr.md (hunk 2 above)
Comment (flibustier) [inline README-fr.md]: That's better! 🙆‍♂️

Comment (Sinequanonh) [inline README-fr.md]: Another thing, if you want foreigners to have a better look at it, you might want to change from `clef`🗝  to `clé`🔑 which is its "modern" version

Comment (flibustier) [inline README-fr.md]: I prefer clef but it's personnal, seems legit!

**Comment by netcode** [comment]
Awesome job , Thanks

**Pull request merged**